Dockerfile.release

```diff
@@ -0,0 +1,8 @@
+FROM gcr.io/distroless/static
+
+# Copy the binary that goreleaser built
```

Comment:
Is this the pattern we want to start using? `Dockerfile` + `Dockerfile.release` or should we just update `Dockerfile` to `Dockerfile.release`'s content?